test_runner/regress/test_wal_acceptor.py

```diff
@@ -1406,11 +1405,7 @@ def __init__(
     ):
         self.repo_dir = repo_dir
         self.port_distributor = port_distributor
-        self.broker = NeonBroker(
-            logfile=Path(self.repo_dir) / "storage_broker.log",
-            port=self.port_distributor.get_port(),
-            neon_binpath=neon_binpath,
-        )
+        self.fake_broker_endpoint = f"http://127.0.0.1{port_distributor.get_port()}"
```

Comment:
Missing a `:`

```suggestion
        self.fake_broker_endpoint = f"http://127.0.0.1:{port_distributor.get_port()}"
```

Reply:
Is it the case that for `SafekeeperEnv` we don't need a real broker?

Reply:
good catch on the missing `:`.

> Is it the case that for SafekeeperEnv we don't need a real broker?

My understanding is that this test doesn't need a pageserver (judging by name `test_safekeeper_without_pageserver`).

No need for pageserver = no need for broker.

Reply:
I'll ask for post-merge review from Arseny, but, don't want to hold up this PR while he's on vacation